Lib/venv/scripts/nt/activate.bat

```diff
@@ -15,8 +15,8 @@ if not defined PROMPT set PROMPT=$P$G
 if defined _OLD_VIRTUAL_PROMPT set PROMPT=%_OLD_VIRTUAL_PROMPT%
 if defined _OLD_VIRTUAL_PYTHONHOME set PYTHONHOME=%_OLD_VIRTUAL_PYTHONHOME%
 
-set _OLD_VIRTUAL_PROMPT=%PROMPT%
-set PROMPT=(__VENV_PROMPT__) %PROMPT%
+set "_OLD_VIRTUAL_PROMPT=%PROMPT%"
```

Comment:
The form `set "A=B"` looks a little odd, I would have expected `set A="B"` - does this latter form not work?

Reply:
Yeah, it’s one of those odd batch-specific things. Here’s an example of this behavior from my terminal:

```batch
> set foo="X"

> echo %foo%
"X"

> set "bar=X"

> echo %bar%
X
```

Someone on StackOverflow asking the same question as you: <https://stackoverflow.com/questions/535975/dealing-with-quotes-in-windows-batch-scripts>.
Documentation for this behavior: <https://ss64.com/nt/set.html>.